### PR TITLE
1339 - IdsTabs alabaster cleanup

### LINKED ISSUE #1339: Tabs: Alabaster Cleanup
**Describe the bug**
Need to finish up alabaster theme. on the tab component.

- Module tabs -> https://main-enterprise.demo.design.infor.com/components/tabs-module/example-index.html
- Header -> https://main-enterprise.demo.design.infor.com/components/header/example-index.html
- ~- Multi Tabs -> https://main-enterprise.demo.design.infor.com/components/tabs-multi/example-index.html~
- Header Tabs -> https://main-enterprise.demo.design.infor.com/components/tabs-header/example-index.html
- Vertical Tabs -> https://main-enterprise.demo.design.infor.com/components/tabs-vertical/example-index.html

https://www.figma.com/file/gRpli2TgQ3scLE6GF4e2Hz/Alabaster-Clean-up?type=design&node-id=127-38900&t=bYNV1tThUhGQXib7-0

**Expected behavior**
Add design changes, adapt personalization colors and add deprecated label to multi tabs page

### PR DESCRIPTION
**Pull request by jdilag90**
**Explain the details for making this change. What existing problem does the pull request solve?**
Adds alabaster changes to tab variants
Here's the [figma](https://www.figma.com/file/gRpli2TgQ3scLE6GF4e2Hz/Alabaster-Clean-up?type=design&node-id=127-38900&mode=design&t=dpqYzczH4cRrCmdA-0)

**Related github/jira issue (required)**:
Fixes #1339 

**Steps necessary to review your pull request (required)**:
1. Checkout branch, `npm install`, `npm run start`
2. Go to each http://localhost:4300/ids-tabs examples
3. Test tab styles, by clicking, hovering, keyboard navigating/Tabbing
4. Test each page in light, dark, contrast mode

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

**Review by tmcconechy — CHANGES_REQUESTED** [review]
Just a few things but overall looks excellent.

http://localhost:4300/ids-tabs/app-example.html
- for disabled color a) keep the background color the same as normal tabs b) make the text slate 50 (was decided in QA on other issue)
- resize the page so the more button shows, then click + or reload and see the tabs change height
- hover state on the X should a) go up in icon color one shade b) have 8px border-radius c) should use  background-color: rgba(0,0,0,.1) instead of azure
Screenshot 2023-07-19 at 12 05 01 PM
- text should go to ellipsis when cut 
Screenshot 2023-07-19 at 12 05 46 PM
- can all the module tabs get X? i think thats a setting right?

http://localhost:4300/ids-tabs/counts.html
- also the x should just change the icon glow, should a) go up in icon color one shade b) have 8px border-radius c) should use  background-color: rgba(0,0,0,.1) instead of azure

http://localhost:4300/ids-tabs/dismissible.html
- same with hover glow

http://localhost:4300/ids-tabs/header-tabs.html
- can you check the height it looks combined a lot more https://main-enterprise.demo.design.infor.com/components/tabs-header/example-index.html?colors=ffffff

http://localhost:4300/ids-tabs/vertical.html
- think it needs a max width and then text ellipsis (arbitrary max for now maybe 250) - can see this if you make the text very large (can maybe use ` <ids-text overflow="ellipsis" tooltip="true">`

Did you check through all these a bit?

https://www.figma.com/file/gRpli2TgQ3scLE6GF4e2Hz/Alabaster-Clean-up?type=design&node-id=137-31812&mode=design&t=sdx2fkFQhryVNv52-0 

https://www.figma.com/file/gRpli2TgQ3scLE6GF4e2Hz/Alabaster-Clean-up?type=design&node-id=527-33769&mode=design&t=yBF5DWr1k7WBWJo3-0

https://www.figma.com/file/gRpli2TgQ3scLE6GF4e2Hz/Alabaster-Clean-up?type=design&node-id=478-32401&mode=design&t=XGH0gcwpDJhJ1Gjv-0

A bit hard to follow bit maybe worth a second look

**Comment by tmcconechy** [comment]
Same here with the change log in beta.14 please @jdilag90

**Review by tmcconechy — DISMISSED** [review]

**Review by EdwardCoyle — DISMISSED** [review]
